app/models/searchable.rb

```diff
@@ -148,7 +148,16 @@ def search_index
     end
   end
 
+  # TODO: I don't think this is a very nice way of achieving this behaviour
```

Comment:
I wonder how this works for organisations, as these have translations but aren't included in search?

Reply:
I've come up with a solution that doesn't seem to affect any of the other document types.  I'm not sure why we're not filtering for other documents 😕 .